crates/sui-framework/sources/governance/sui_system.move

```diff
@@ -429,6 +432,137 @@ module sui::sui_system {
         vec_set::remove(reporters, &sender);
     }
 
+    // ==== validator metadata management functions ====
+
+    /// Update a validator's name.
+    public entry fun update_validator_name(
+        self: &mut SuiSystemState,
+        name: vector<u8>,
+        ctx: &TxContext,
+    ) {
+        let self = load_system_state_mut(self);
+        let validator = validator_set::get_active_or_pending_validator_mut(&mut self.validators, ctx);
+        validator::update_name(validator, string::from_ascii(ascii::string(name)));
```

Comment:
Not sure if this decision has already been made or not, but do we want to require that a validator's name (and description) be valid ASCII or do we want to support valid UTF8 for these fields? @emmazzz what are your thoughts on this?

Reply:
I think we prefer to keep it as ascii for simplicity for now. If this is a strongly desired feature by validators, we could support this post mainnet.